### ISSUE TITLE
Error in timefrequency with plotStokes

### ISSUE THREAD

**Issue by SRimbaud** [jflamant/bispy#1]
```python

import quaternion  # load the quaternion module
import numpy as np

#if BiSPy is not located in your python module directory
#import sys
#sys.path.append('/BiSPy_location_path/')
import bispy.bispy as bsp #My bispy module is in a folder named bispy.
pi = np.pi #Added 
N = 1024 # length of the signal

#linear chirps constants
a = 250*pi
b = 50*pi
c = 150*pi
#time vector
t = np.linspace(0, 1, N)

#first chirp
theta1 = pi/4 #constant orientation
chi1 = pi/6-t #reversing ellipticity
phi1 = b*t+a*t**2 #linear chirp

#second chirp
theta2 = pi/4*10*t #rotating orientation
chi2 = 0 #constant null ellipticity
phi2 = c*t+a*t**2 #linear chirp

#common amplitude -- simply a window
env = bsp.utils.windows.hanning(N)
#define chirps x1 and x2
x1 = bsp.utils.bivariateAMFM(env, theta1, chi1, phi1)
x2 = bsp.utils.bivariateAMFM(env, theta2, chi2, phi2)

#sum it
x = x1 + x2
fig, ax = bsp.utils.visual.plot3D(t, x)
fig.show()
window = bsp.utils.windows.hanning(101)
spacing = 1

S = bsp.timefrequency.QSTFT(t, x, window, spacing)

fig, ax = S.plotStokes()
fig.show()
```

Error at S.plotStokes() : 
```python
File "./bispy/timefrequency.py" line 262, in plotStokes
    im0 = ax[0].imshow(self.S1n[:self.Nfft / 2, :], origin='lower', interpolation='none', aspect='auto',cmap=cmap, extent=[self.t.min(), self.t.max(), 0, self.f[self.Nfft / 2-1]], vmin=-1, vmax=+1)
TypeError: slice indices must be integers or None or have an __index__ method
```
I'm working with python 3.6.1.
Extracted from : [tutorial](https://bispy.readthedocs.io/en/latest/tutorials/timefrequency.html#quaternion-short-term-fourier-transform-q-stft-example)

**Comment by SRimbaud** [comment]
Same Error in the [tutorial](https://bispy.readthedocs.io/en/latest/tutorials/timefrequency.html#quaternion-short-term-fourier-transform-q-stft-example) script 
with : 
```python
S.extractRidges()

```
return : 
```python 

File "./bispy/timefrequency.py", line 236, in extractRidges
    self.ridges = _extractRidges(self.S0[:self.Nfft / 2, :], parThresh, parMinD)
TypeError: slice indices must be integers or None or have an __index__ method
```

**Comment by jflamant** [comment]
Corrected by 6fb73c8cbc5a85361eb3a07a7f228a38498c1e21.
